### ISSUE TITLE
Hand smoothing settings appear to be non-functional

### ISSUE THREAD

**Issue by LexiBasilisk** [Yellow-Dog-Man/Resonite-Issues#1691]
### Describe the issue.

While testing the new hand smoothing settings, I observed that neither the positional or rotational smoothing seemed to take effect, even when their values were maxed out.

### To Reproduce

- Enable the hand smoothing sliders for position/rotation

### Expected behavior

Tracked controllers/hands should be smoothed out by these settings.

### Screenshots

https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/18a1855d-7b90-4d88-bf2a-5d6ce283e61f

### Resonite Version Number

2024.4.15.1234

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Oculus Quest 1

### Log Files

[SHIFTY-LAPTOP - 2024.4.15.1234 - 2024-04-15 13_58_57.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14984360/SHIFTY-LAPTOP.-.2024.4.15.1234.-.2024-04-15.13_58_57.log)

### Additional Context

I had attempted to mitigate the tracking jitter of my Quest 1's touch controllers using these settings, and observed that these settings seem to have no effect, regardless of the value they are set to.

### Reporters

@shiftyscales

**Comment by RyuviTheViali** [comment]
Strange, hand smoothing (and the other smoothing settings) works properly for me.

**Comment by ohzee00** [comment]
Does this happen with other avatars? Could this avatar in particular have a interesting hand proxy setup?

**Comment by Frooxius** [comment]
This should be addressed in 2024.4.17.1420. Can you test if this works now?

**Comment by Krzeszny** [comment]
It also does nothing for me. Not sure if it matters what I'm using.
https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/10529134/68eaba6c-267a-4ff3-9c9d-15c9eea3f808
[DESKTOP-VHA2V7H - 2024.6.11.74 - 2024-06-12 12_42_11.log](https://github.com/user-attachments/files/15803508/DESKTOP-VHA2V7H.-.2024.6.11.74.-.2024-06-12.12_42_11.log)

**Comment by shiftyscales** [comment]
> It also does nothing for me, but for all trackers. Should I open a new issue?

Based on the log file provided- it appears you are using mods, @Krzeszny. Please retest without and see if the issue persists.

That sounds like it is probably a separate issue, however. What brand of tracker are they? Are they detected in SteamVR?

Do the trackers register at all inside of Resonite / do they track properly otherwise aside from not being smoothed?

You may be experiencing #1874.

**Comment by Krzeszny** [comment]
> > It also does nothing for me, but for all trackers. Should I open a new issue?
> 
> Based on the log file provided- it appears you are using mods, @Krzeszny. Please retest without and see if the issue persists.
The issue persists.
> That sounds like it is probably a separate issue, however. What brand of tracker are they? Are they detected in SteamVR?
I'm using MixedVR for tracker emulation from Oculus Rift CV1 in SteamVR. They work normally as any other SteamVR trackers though.
> Do the trackers register at all inside of Resonite / do they track properly otherwise aside from not being smoothed?
They work normally besides not being smoothed.
> You may be experiencing #1874.
Nope, I've just checked in BodyNode and feet trackers have smoothing set to 100 while hip has smoothing at -1, but none of them are visibly smoothed.

**Comment by shiftyscales** [comment]
Lower values are more smoothed, @Krzeszny. If you have them all at 100- that effectively means there is no smoothing applied. Try a value around 10 or less.

**Comment by Krzeszny** [comment]
> Lower values are more smoothed, @Krzeszny. If you have them all at 100- that effectively means there is no smoothing applied. Try a value around 10 or less.

Oh, so it is backwards, but it works. So my issue was bad UI. Even in the menu in Devices, 100% means no smoothing and 0% means maximum smoothing.

**Comment by kulzae** [comment]
This isn't the first time I have seen feedback of the values being "backwards" I get it's hooking to something that's expecting that input but perhaps the UI could be reworked to go from no smoothing to a reasonable maximum instead?
